README.md

```diff
@@ -43,8 +43,10 @@ In particular,
 - Thinking beyond simple circuits will allow us to do new things and progress the field.
 
 Compilation and optimization of such programs is crucial to support quantum hardware execution,
-however current de-facto interchange formats (such as QIR and OpenQASM) either aren't designed for quantum optimization — lacking the explicit
-structure needed for advanced high-level optimizations — nor tool-agnostic exchange, locking developers into particular toolchains like LLVM.
+however current de-facto interchange formats (such as QIR and OpenQASM) either aren't designed for
+quantum optimization — lacking the explicit structure needed for advanced high-level
+optimizations — nor tool-agnostic exchange, locking developers into particular toolchains like
+LLVM.
 
 ***This creates a bottleneck for collaboration and makes it difficult to develop and share
```

Comment:
[I can't comment on the collapsed lines (54-56), so leaving a comment here.]  

It would be helpful to provide some more motivation as to why a serialized, in-memory representation is useful (presumably w.r.t. speed and scale?). 

EDIT: I see this is covered in the FAQs but putting it up front wouldn't hurt.

Reply:
I might address this in a subsequent PR!